abusech-feodo/package.yaml

```diff
@@ -22,22 +22,3 @@ contexts:
     type: lookup-table
     description: A table that contains the elements of the feodo IP blocklist.
 
```

Comment:
```suggestion
```
It seems all `package.yaml` files that contained examples now have a trailing empty line. Please remove that as well.

Reply:
FYI, you replaced 2 newlines with 0. Every file should end with exactly 1 newline.